### ISSUE TITLE
LDRS 501 Edits

### ISSUE THREAD

**Issue by kmarjanovic** [TWUOnline/bug-tracking#924]
**Which course contains the bug?**  
[LDRS 501 (6 week MASTER - 2023)
](https://learn.twu.ca/course/view.php?id=28347&section=7#tabs-tree-start) 
 
**Describe the error and/or your suggestion**  
- [x] Please add the missing Assignment  
Individual Assignment 1 - Personal Strategic Assessment
See instructions and criteria [here](https://learn.twu.ca/course/view.php?id=15738&section=2#tabs-tree-start) in the Assessment section of Unit 1
- [x]  Please fix the formatting so it is the same as the [template](https://learn.twu.ca/course/view.php?id=30164&section=1)
  - [x] Numbering
  - [x] Activity checklist formatting
  - [x] Learning Activities should be in a separate page
  - [x] Take out assessment info from Overview section in Moodle Book, as well as at the end of the unit
- [x] Check unit 4 topic Strategic Team Review- Toolkit.  The video is embedded twice.  Is this about the multimedia settings for links?
- [x] Unit 5 seems to have Unit 9 content.  Please check [FAR master](https://learn.twu.ca/course/view.php?id=15738&section=6#tabs-tree-start)

Just need to confirm whether Units 7-9 were intentionally cut from the course.

**Comment by MeeksonHundoo** [comment]
Hello @kmarjanovic I've formatted this course to the MA Lead 6 week course template. I have not edited anything on the moodle course book apart from putting 2 learning activities in week 2 and making "left brain vs right brain functions" as a subtopic per our discussion. Also, I've embedded the missing H5P activity of the Q sort sheet in the learning activity. 

With regards to unit 5 having unit 9 contents, it seems so to me as well. But I have left it unchanged as Mark or the SME might have made those changes. Few other observations that I had while formatting this was, that the "Implicit Association Test" is repeating 2x in the moodle books (week 4 & week 5 LA's) and the "Q sort sheet H5P activity" is repeating in week 5 and Week 6 LA's.
